scripts/update_appup.escript

```diff
@@ -2,94 +2,97 @@
 %% -*- erlang-indent-level:4 -*-
 
 usage() ->
-"A script that fills in boilerplate for appup files.
-
-Algorithm: this script compares md5s of beam files of each
-application, and creates a `{load_module, Module, brutal_purge,
-soft_purge, []}` action for the changed and new modules. For deleted
-modules it creates `{delete_module, M}` action. These entries are
-added to each patch release preceding the current release. If an entry
-for a module already exists, this module is ignored. The existing
-actions are kept.
-
-Please note that it only compares the current release with its
-predecessor, assuming that the upgrade actions for the older releases
-are correct.
-
-Note: The defaults are set up for emqx, but they can be tuned to
-support other repos too.
-
-Usage:
-
-   update_appup.escript [--check] [--repo URL] [--remote NAME] [--skip-build] [--make-commad SCRIPT] [--release-dir DIR] <previous_release_tag>
-
-Options:
-
-  --check            Don't update the appfile, just check that they are complete
-  --repo             Upsteam git repo URL
-  --remote           Get upstream repo URL from the specified git remote
-  --skip-build       Don't rebuild the releases. May produce wrong results
-  --make-command     A command used to assemble the release
-  --prev-release-dir Previous version's release dir (if already built/extracted)
-  --release-dir      Release directory
-  --src-dirs         Directories where source code is found. Defaults to '{src,apps,lib-*}/**/'
-".
-
--record(app,
-        { modules       :: #{module() => binary()}
-        , version       :: string()
-        }).
+    "A script that fills in boilerplate for appup files.\n"
```

Comment:
Ugh, erlfmt made an unreadable mess here. Add `%% erlfmt-ignore` to this function?